tembo-operator/src/stacks/templates/mongo_adapter.yaml

```diff
@@ -0,0 +1,78 @@
+name: MongoAdapter
+description: Document-facing workloads on Postgres.
+image: "quay.io/tembo/standard-cnpg:15.3.0-1-1096aeb"
+stack_version: 0.1.0
+appServices:
+  - name: fdb-api
+    image: ghcr.io/ferretdb/ferretdb
+    routing:
+      - port: 27018
+        ingressPath: /ferretdb/v1
+        entryPoints:
+          - ferretdb
+        ingressType: tcp
+    env:
+      - name: FERRETDB_POSTGRESQL_URL
+        valueFromPlatform: ReadWriteConnection
+      - name: FERRETDB_LOG_LEVEL
+        value: debug
+      - name: FERRETDB_STATE_DIR
+        value: '-'
+      - name: FERRETDB_LISTEN_TLS_CERT_FILE
+        value: /tembo/certs/tls.crt
+      - name: FERRETDB_LISTEN_TLS_KEY_FILE
+        value: /tembo/certs/tls.key
+      - name: FERRETDB_LISTEN_TLS
+        value: :27018
+    storage:
+      volumes:
+        - name: ferretdb-data
+          ephemeral:
+            volumeClaimTemplate:
+              spec:
+                accessModes:
+                  - ReadWriteOnce
+                resources:
+                  requests:
+                    storage: 1Gi
+        volumeMounts:
+          - name: ferretdb-data
+            mountPath: /state
```

Comment:
it would be nice to not need this (because it would make the container much faster to move across servers) - and I thought we found a way to not include it? In any case, that can be a follow up or maybe it's needed for some reason